ctru-rs/src/services/romfs.rs

```diff
@@ -1,30 +1,57 @@
-//! Read-Only Memory FileSystem
+//! Read-Only Memory FileSystem service.
+//!
+//! This service lets the application access a virtual mounted device created using a folder included within the application bundle.
+//! After mounting the RomFS file system, the included files and folders will be accessible exactly like any other file, just by using the drive prefix `romfs:/<file-path>`.
```

Comment:
I wonder if it's worth mentioning the weirdness with `std::path` (i.e. https://github.com/rust-lang/rust/issues/52331) somewhere in here, or maybe in the `fs` module. I guess we at least have some examples and _most_ of the time people shouldn't have to worry about it (just pass their paths in plainly).